core.js

```diff
@@ -3854,6 +3854,7 @@ const _makeWindow = (options = {}, parent = null, top = null) => {
     }
   };
   // window.Buffer = Buffer; // XXX non-standard
+  window.G = global;
```

Comment:
What does this have to do with gamepads?

Reply:
Whoops. I use that for debugging. I’ll nuke it.

It’s so handy that you might want to flag it.